packages/static-wado-plugins/jest.config.js

```diff
@@ -0,0 +1,10 @@
+const baseConfig = require("../../.config/jest/jest.config");
+
+module.exports = {
+  ...baseConfig,
+  // just does not yet support subpath export from node
+  moduleNameMapper: {
```

Comment:
if this project needs this mapping it means we might need to move the configuration for baseConfig.
This might be true considering we dont consume generated built files from packages but the source code instead

Reply:
Agreed, done.